dotnet/src/SemanticKernel/CoreSkills/TextSkill.cs

```diff
@@ -2,6 +2,8 @@
 
 using Microsoft.SemanticKernel.SkillDefinition;
 
+#pragma warning disable CA1308
```

Comment:
I'm adding this disable here because those skills are intended for LLM interfacing usage only.